src/main/utils/utils.ts

```diff
@@ -10,6 +10,8 @@ export class Utils {
     // 1 GB
     private static readonly MAX_SIZE_EXTRACTED_ZIP_BYTES: number = 1000000000;
     private static readonly COMPRESSION_THRESHOLD_RATIO: number = 100;
+    // week
```

Comment:
Comments should start with capital letters
```suggestion
    // 1 Week
```